### PR TITLE
Merge master to dev

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -84,21 +106,6 @@
...
-* compability for accessing status enums for users ([4b377e8](https://github.com/modelon-community/impact-client-python/commit/4b377e82b29faab98d7d9f04614416f4949ca7fa))
-
-## [2.2.1](https://github.com/modelon-community/impact-client-python/compare/v2.2.0...v2.2.1) (2022-06-09)
-
-
-### Bug Fixes
-
-* added types-requests package as a dependency ([b165fed](https://github.com/modelon-community/impact-client-python/commit/b165feda8c1b20ea415cef72bf39214175343fd8))
-* update python dependencies ([e4ae2d4](https://github.com/modelon-community/impact-client-python/commit/e4ae2d434de0fd458dcabcf4c1d7a6797b68cdd3))
-
```
Comment (efredriksson-modelon) [inline CHANGELOG.md:101]: Should these be removed?

**Review by efredriksson-modelon — CHANGES_REQUESTED** [review]
One thing that looks strange with the merge 🔬

File: CHANGELOG.md (hunk 1 above)
Comment (abhilash-kumar-nair) [reply CHANGELOG.md]: Fixed it 😄

**Review by efredriksson-modelon — APPROVED** [review]
👍

**Pull request merged**